apps/system/style/chrome/chrome.css

```diff
@@ -1,3 +1,10 @@
+/**
+ * Force default alignment even when elements have a dir attribute or
+ * unicode-bidi: plaintext
+ */
+html[dir="ltr"] { text-align: left; }
+html[dir="rtl"] { text-align: right; }
```

Comment:
lets scope these selectors to something more specific to the url bar and browser chrome, to avoid the possibility of a bunch of fallout we don't have time to deal with right now